solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -255,10 +255,11 @@ public String toString() {
   private final Map<String, String> replicasInLeaderInitiatedRecovery = new HashMap<>();
 
   // This is an expert and unsupported development mode that does not create
```

Comment:
IMO we should drop this entirely.  I've never heard of it and doesn't really sound useful TBH.  Why run Solr that only runs ZK (that is confusing BTW!) when one could run ZK via Docker?  Docker makes running anything easy in a developer workstation scenario, which is what this feature here is for.

CC @markrmiller

Reply:
Done in #1225833196dea24f413abbc3cc3fbef3d5b769fb